<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="game_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>315</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Tic Tac Toe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_with_bot_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>315</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Play with bot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_with_player_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>315</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Play with player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_online_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>315</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Play online</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
